spec/factories/profile.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :profile do
+    title {Faker::Name.name}
+    position {Faker::Job.position}
+    address {Faker::Address.city}
+    email {Faker::Internet.email}
+    phone {"0123456"}
```

Comment:
faker đâu e

Reply:
faker Số điện thoại toàn trên 10 ký tự thôi anh ạ mà Việt Nam mình h toàn dung 10 số nên em fix cứng ạ